pkg/filter/driftignore.go

```diff
@@ -11,14 +11,16 @@ import (
 )
 
 type DriftIgnore struct {
-	resExclusionList   map[string]struct{} // map[type.id] exists to ignore
-	driftExclusionList map[string][]string // map[type.id] contains path for drift to ignore
+	resExclusionList         map[string]struct{} // map[type.id] exists to ignore
+	resExclusionWildcardList map[string]struct{} // map[type.id] exists with wildcard to ignore
```

Comment:
It works well but I wonder if we should have a list of all wildcards patterns. What if we want to handle another glob pattern like `{a,b,c}` ?

Reply:
Great comment, i'll be glad to work on this in another pr.

My opinion on the subject would be to refactor the whole thing a bit to predict this behaviour, or else to assume a passage in strMatchChecker for each resource/exclusion (since the function also acts as a strcmp if there is no wildcard).

Simply, calls with string copy at each recursion (pointer arithmetic unavailable) seemed expensive and avoidable with a first check on a list containing only strings, which is also possible to continue to fill with a layer in addition to checking in readDriftIgnoreLine if a glob is present in the line buffer.

Reply:
we could also use the same list and adapt the check when/if we add new patterns. When we discussed this it looked like a good idea to preprocess like the other list but it might need to change if the usage change later